contrib/cmake/CMakeLists.txt

```diff
@@ -469,14 +469,36 @@ endif(USE_THIRDPARTY_FROM_VCPKG)
 # Migration from autotools note: there was a check that determined if libmediainfo was compiled with UNICODE.
 set(UNICODE 1)
 
+# Prepare to test for DriveNotify support
+unset(HAVE_DRIVE_NOTIFY CACHE)
+
 if(WIN32)
+
+    # Test for WMI / WBEM / DriveNotify support
+    include(CheckCXXSourceCompiles)
+    set (CMAKE_REQUIRED_LIBRARIES "wbemuuid")
+    check_cxx_source_compiles("
+    #include <wbemcli.h>
+    int main()
```

Comment:
Hi, for the building of release software, we really want to know that all expected features are present.  I suggest using the normal USE_DRIVE_NOTIFY cached variable, which the person configuring the build would set.   If they choose to turn that on, the build should fail if it's not present.  And if they choose not to use it, it doesn't matter if the headers etc are present or not.   We wouldn't want to accidentally release software that we thought contained this feature but the compile system had silently skipped.  thanks!

Reply:
Ok, replaced `HAVE_DRIVE_NOTIFY` with `USE_DRIVE_NOTIFICATIONS`.

Reply:
Does the same logic applies to the building with qmake? Please, @catalinr-m, make sure ;)

Reply:
Sorry, which logic are you asking about?
The build test was removed, and `HAVE_DRIVE_NOTIFY` -> `USE_DRIVE_NOTIFICATIONS` change was done in `qt\sdk.pri` too (though in a later commit).

Reply:
Yes, I see the `sdk.pri` contains conditional code depending on the `USE_DRIVE_NOTIFICATIONS` is defined or not. However, while I can see that you set that config [here](https://github.com/meganz/sdk/pull/2441/files#diff-7851744e91538f5741554ee3a428b0901d30dce2c98dea27272330a817259f84R45) for building with cmake, I don't see anything analog for qmake+autotools. I guess that it needs to touch the `configure.ac` in a way that it defines it in the same way than for building with CMake. By doing so, Jenkins will start building + testing, since currently it's not building with `USE_DRIVE_NOTIFICATIONS`.

Reply:
Another option would be to hardcode the option in the `sdk.pri`, but I think that's not the right way (probably not for cmake either), since we don't want to build the SDK with this feature for all targets.